### PR TITLE
Adds an XPath expression testing UI

### LINKED ISSUE #104: Add XPath evaluation UI

### PR DESCRIPTION
**Pull request by richardrodgers**
closes #104

### CODE REVIEW

**Comment by JPrevost** [comment]
Hey, looks like I managed to keep Coveralls from reporting as part of the build status.

**Comment by richardrodgers** [comment]
Cool - I like to be informed of coverage deltas, but don't necessarily want to block deployment for it

File: app/views/static/sandbox.scala.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+@*****************************************************************************
+ * Page for testing XPath/XQuery expressions against target resources        *
+ * Useful for authoring Finders                                              *
+ * Copyright (c) 2015 MIT Libraries                                          *
+ *****************************************************************************@
+@(sbForm: Form[(String, String)], results: List[String])@*(implicit hubContext: HubContext)*@
+@import helper._
+@layout.main("Sandbox - TopicHub") {
+  <div class="container">
```
Comment (JPrevost) [inline app/views/static/sandbox.scala.html]: Changing this to `<div class="container-fluid">` prevents it from sliding behind the top navigation.

Comment (JPrevost) [inline app/views/static/sandbox.scala.html]: Never mind. Something got introduced that is causing that in Master as well. I'll debug and open a ticket.

File: app/views/static/sandbox.scala.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <div class="row">
+      <div class="col-md-4">
+      @form(routes.Application.testExpression()) {
+        @textarea(sbForm("expression"), 'rows -> 8, 'cols -> 40)
+        @inputText(sbForm("resourceUrl"), 'size -> 50)
+        <input type="submit" value="Test Expression">
+      }
+      <br/>
+      <a href="@routes.Application.sandbox" class="btn btn-primary">Start Over</a>
```
Comment (JPrevost) [inline app/views/static/sandbox.scala.html]: From a ux perspective, this is confusing as heck to have the primary button reset the form. That said, I don't think that matters for this form so I'd leave it like this for this PR.

File: app/workers/Cataloger.scala [hunk 3]
```diff
@@ -296,6 +298,34 @@ object Cataloger {
     //item.changeState("cataloged")
   }
 
+  def testExpression(expr: String, exprType: String, source: String) = {
+    val srcStream = new URL(source).openConnection.getInputStream
+    val results = exprType match {
+      case "XPath" => testXPathExpression(expr, srcStream)
+      //case "XQuery" => testXQueryExpression(expr, srcStream)
```
Comment (JPrevost) [inline app/workers/Cataloger.scala]: Oooh. Yes please :)

**Comment by JPrevost** [comment]
:+1:

**Pull request merged**